sdk/python/kfp/components/_component_store.py

```diff
@@ -19,7 +19,16 @@
 
 
 class ComponentStore:
-    def __init__(self, local_search_paths=None, url_search_prefixes=None, auth=None):
+    def __init__(
+        self, local_search_paths=None, url_search_prefixes=None, auth=None):
+        """Instantiates a ComponentStore.
+
+        Feature stage: Stable. This feature is well tested and won't be dropped.
```

Comment:
Is it possible to make the message more compact so that the documentation is easier to read? Maybe

```
Feature stage: [Stable](
https://github.com/kubeflow/pipelines/blob/07328e5094ac2981d3059314cc848fbb71437a76/docs/release/feature-stages.md#stable).
```

Reply:
Thanks. Done.